src/clients/c++/experimental_api_v2/library/grpc_client.h

```diff
@@ -299,17 +301,65 @@ class InferenceServerGrpcClient : public InferenceServerClient {
           std::vector<const InferRequestedOutput*>(),
       const Headers& headers = Headers());
 
+  /// Starts a grpc bi-directional stream to send streaming inferences.
+  /// \param callback The callback function to be invoked on receiving a
+  /// response at the stream.
+  /// \param skip_stats Indicates whether client library should record the
+  /// the client-side statistics for inference requests on stream or not.
+  /// The library currently doesn't support statistics for decoupled
+  /// streaming. Set this option True when there is no 1:1 mapping between
+  /// request and response on the stream.
+  /// \param headers Optional map specifying additional HTTP headers to
+  /// include in the metadata of gRPC request.
+  /// \return Error object indicating success or failure of the request.
+  Error StartStream(
+      OnCompleteFn callback, bool skip_stats = false,
```

Comment:
Can you make this "enable_stats" instead and set the default to true.

Reply:
Done